src/Compilers/CSharp/csc/project.json

```diff
@@ -1,6 +1,6 @@
 ﻿{
   "dependencies": {
-    "Microsoft.DiaSymReader.Native": "1.5.0"
+    "Microsoft.DiaSymReader.Native": "1.6.0-beta2-25219"
```

Comment:
I'm assuming that we will not ship dev15.3 with a reference to beta package. Can you file a follow-up issue to fix this reference once a release package is available? 
It's probably also good to think about timeline to make this happen.